src/core/flatbuffers_test.cc

```diff
@@ -3,6 +3,11 @@
 //
 
 #include <absl/strings/escaping.h>
+
+#ifndef __USE_GNU  // needed to flatbuffers to compile with musl libc.
+#define FLATBUFFERS_LOCALE_INDEPENDENT 0
+#endif
+
```

Comment:
Nit: consider having our own include file to declare this and then include flatbuffers.

Reply:
good idea